modules/ROOT/pages/questions/question-14.adoc

```diff
@@ -0,0 +1,9 @@
+14. Inferring from
```

Comment:
@lnewson 

FYI: https://www.tiny.cloud/docs/api/tinymce.html/tinymce.html.schema/

Reply:
I think Mike has answered this already. There's also some details related to this in my reply to Q13.